install/kubernetes/helm/istio-remote/values.yaml

```diff
@@ -45,10 +45,6 @@ global:
     # Default setting for service-to-service mtls. Can be set explicitly using
     # destination rules or service annotations.
     enabled: false
-    # List of fully qualified services to exclude from mtls
-    # TODO: add the templating.
-    mtlsExcludedServices:
-    - "kubernetes.default.svc.cluster.local"
```

Comment:
One question, if we removed this parameter, how to handle the service of `kubernetes.default.svc.cluster.local`? Can the service still be accessed when mtls enabled?